### PR TITLE
Update nodejs context

### PR DESCRIPTION
**Pull request by sadiqevani**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

> Node.js is not a programming language, JavaScript is, took the definition from https://nodejs.org/en/

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/javascript/index.md [hunk 1]
```diff
@@ -10,4 +10,4 @@ topic: javascript
 url: https://developer.mozilla.org/en-US/docs/Web/JavaScript
 wikipedia_url: https://en.wikipedia.org/wiki/JavaScript
 ---
-JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. Node.js is a programming language running on the V8 JavaScript execution engine. JS also brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
+JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. JS also brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
```
Comment (kenyonj) [inline topics/javascript/index.md]: Your edit makes sense, but now I am wondering why there is anything about `Node.js` in the description of the `JavaScript` topic at all?

I think I'd vote to remove that sentence altogether and add `nodejs` as a related topic (in my other suggestion).

```suggestion
JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. JS brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
```

File: topics/javascript/index.md [hunk 2]
```diff
@@ -10,4 +10,4 @@ topic: javascript
 url: https://developer.mozilla.org/en-US/docs/Web/JavaScript
```
Comment (kenyonj) [inline topics/javascript/index.md]:
```suggestion
url: https://developer.mozilla.org/en-US/docs/Web/JavaScript
related: nodejs
```

**Comment by sadiqevani** [comment]
Hey @kenyonj, I agree with your changes, though the description does not encapsulate what "JavaScript" is, what if we put the following text:

```
JavaScript (JS) is a lightweight interpreted or JIT-compiled programming language with first-class functions. While it is most well-known as the scripting language for Web pages, many non-browser environments also use it, such as Node.js, Apache CouchDB and Adobe Acrobat. JavaScript is a prototype-based, multi-paradigm, dynamic language, supporting object-oriented, imperative, and declarative (e.g. functional programming) styles. Read more about JavaScript.
```
Taken from MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript Though I'm not sure if this would create any legal implications.

File: topics/javascript/index.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ short_description: JavaScript is an object-oriented programming language used al
   HTML and CSS to give functionality to web pages.
 topic: javascript
 url: https://developer.mozilla.org/en-US/docs/Web/JavaScript
+related: nodejs
 wikipedia_url: https://en.wikipedia.org/wiki/JavaScript
 ---
-JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. Node.js is a programming language running on the V8 JavaScript execution engine. JS also brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
+JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. JS brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
```
Comment (kenyonj) [inline topics/javascript/index.md]: @sadiqevani moving your comment here:
> Hey @kenyonj, I agree with your changes, though the description does not encapsulate what "JavaScript" is, what if we put the following text:

> JavaScript (JS) is a lightweight interpreted or JIT-compiled programming language with first-class functions. While it is most well-known as the scripting language for Web pages, many non-browser environments also use it, such as Node.js, Apache CouchDB and Adobe Acrobat. JavaScript is a prototype-based, multi-paradigm, dynamic language, supporting object-oriented, imperative, and declarative (e.g. functional programming) styles. Read more about JavaScript.

Comment (kenyonj) [reply topics/javascript/index.md]: I think what you have suggested is great. I would just remove the last sentence: `Read more about JavaScript`, for obvious reasons.

Comment (sadiqevani) [reply topics/javascript/index.md]: @kenyonj done.

**Comment by kenyonj** [comment]
Looks like there is a CI failure @sadiqevani . Please resolve that and we can get this merged.

**Comment by sadiqevani** [comment]
@kenyonj 

The issue seems to be with the short desciption and 130char limit.

I have reduced the description from:

```
JavaScript (JS) is a lightweight interpreted programming language with first-class functions used alongside HTML and CSS to give functionality to web pages.
```

To:

```
JavaScript (JS) is a lightweight interpreted programming language with first-class functions.
```

**Comment by sadiqevani** [comment]
@kenyonj 

I believe this extra information is not relevant / does not impact the context of the sentence: `used alongside HTML and CSS to give functionality to web pages.`

**Pull request merged**